packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -125,11 +125,11 @@ export async function runInterstitialRules<T>(opts: T, rules: InterstitialRule[]
   return signedOut<T>(opts, AuthErrorReason.UnexpectedError);
 }
 
-async function verifyRequestState(options: any, token: string) {
-  const issuer = (iss: string) => iss.startsWith('https://clerk.') || iss.includes('.clerk.accounts');
+async function verifyRequestState(options: any, token: string, issuer = '') {
```

Comment:
❓ Is this relevant to this PR ? Maybe I am missing some context here

Reply:
In the description you are mentioning "refactor verifyToken function"
But is this necessary for this PR ?

Reply:
@panteliselef I updated the description. Yes, this is part of this PR. We need to use the proxyUrl if present during verification since it makes a request to FAPI.

Reply:
@raptisj go it